.changeset/famous-crews-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+"@osdk/foundry-sdk-generator": patch
```

Comment:
General non-blocking question, why are we still using tsup for foundry-sdk-generator and client.unstable?

Reply:
For both, because it does a rollup.

`client.unstable`: because it rolls up and drops all the conjure methods that we don't use, which is an optimization for downstream builds / consumers.
`@osdk/foundry-sdk-generator`: because anything not in `dependencies` nor `peerDependencies` are basically embedded in the output which we can partially leverage (along with my audited dep change) to ensure more stable output and not a repeat of the "magic TS upgrade of Sep2024" bug